src/layers/fp64/arc-layer/arc-layer.js

```diff
@@ -65,7 +65,6 @@ export default class ArcLayer extends Layer {
     this.setState({model});
 
     attributeManager.addInstanced({
-      instancePositions: {size: 4, update: this.calculateInstancePositions},
```

Comment:
instancePositions are not used in 64bit arc layer. it's replaced by instanceTargetPositionFP64 and instanceSourcePositionFP64